app/src/main/java/org/redcross/openmapkit/OSMMapBuilder.java

```diff
@@ -34,8 +34,9 @@
  * nhallahan@spatialdev.com* 
  */
 public class OSMMapBuilder extends AsyncTask<File, Long, JTSModel> {
-    
-    private static final float MIN_VECTOR_RENDER_ZOOM = 18;
+
+    // Render vectors at lower zoom - useful for showing distribution.
+    private static final float MIN_VECTOR_RENDER_ZOOM = 12;
```

Comment:
Let's add this as a setting.

Context for this Nick is we are using this for a malaria spraying project
(hence why we added the color option).  We want to be able to zoom up a
level or two to all the structures in a specific area.  Normally this won't
be more then 100 structures so OMK can handle the rendering at the higher
zoom level.


On Mon, Aug 10, 2015 at 12:22 PM, Nicholas Hallahan <
notifications@github.com> wrote:

> In app/src/main/java/org/redcross/openmapkit/OSMMapBuilder.java
> <https://github.com/AmericanRedCross/OpenMapKit/pull/77#discussion_r36658144>
> :
>
> > @@ -34,8 +34,9 @@
> >   * nhallahan@spatialdev.com*
> >   */
> >  public class OSMMapBuilder extends AsyncTask<File, Long, JTSModel> {
> > -
> > -    private static final float MIN_VECTOR_RENDER_ZOOM = 18;
> > +
> > +    // Render vectors at lower zoom - useful for showing distribution.
> > +    private static final float MIN_VECTOR_RENDER_ZOOM = 12;
>
> We don't want to change this constant, because with typical data sets that
> we have been using in the app, the renderer has trouble at lower zooms, and
> the app may crash.
>
> Perhaps allow this to be set in a settings file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AmericanRedCross/OpenMapKit/pull/77/files#r36658144>.
>